package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "employer-style-base",
-  "version": "2.2.18",
+  "version": "2.2.18-alpha.3",
```

Comment:
@toastercup let's talk about how we want to do this. When should we use `alpha` vs `beta`?